### PR TITLE
Added subsection: 'Dashboard'

### PR DESCRIPTION
**Pull request by Bokaiii**

### CODE REVIEW

File: project_report/tex/design.tex [hunk 1]
```diff
@@ -1 +1,16 @@
 \section{Design}
+
+\subsection{Dashboard design}
+A very important aspect the group has had to concider, is which design pattern
+to model the dashboard after. A design pattern is a form of template, which in
+our case, dictates how the different classes communicate. A relevant design
```
Comment (Kechr) [inline project_report/tex/design.tex]: Personal pronoun "our" -> Remove, please

File: project_report/tex/design.tex [hunk 2]
```diff
@@ -1 +1,16 @@
...
+pattern is the MVC (Model - View - Controller). This pattern separates the
+database (model), the logic (controller) and the GUI (View) into different
```
Comment (Kechr) [inline project_report/tex/design.tex]: Model = data-related logic
View = UI logic
Controller = interface between the above

A more detailed explanation can be found [here](https://www.tutorialspoint.com/mvc_framework/mvc_framework_introduction.htm)

File: project_report/tex/design.tex [hunk 3]
```diff
@@ -1 +1,16 @@
...
+layers, and thereby helps keep separation of concerns. The controller manages
+input from the user, and forwards it to either the model, to retrieve or store
+data, or the view, to present data to the user.
+
+The group has decided to use the MVC design pattern, thus making it possible to
+work on the different aspects of the system (GUI, database, logic) separately.
```
Comment (Kechr) [inline project_report/tex/design.tex:14]: "... different parts of the system ..." 

**Aspect**
     _Noun_: A way in which something can be viewed by the mind.
     _Noun_: A characteristic or feature of something.

File: project_report/tex/design.tex [hunk 4]
```diff
@@ -1 +1,16 @@
...
+To ensure that these different parts of the system can communicate, a 'contract',
+in the form of an interface, should be created.
```
Comment (Kechr) [inline project_report/tex/design.tex]: In this case the controller.

File: project_report/tex/design.tex [hunk 5]
```diff
@@ -1 +1,16 @@
 \section{Design}
+
+\subsection{Dashboard design}
```
Comment (Kechr) [inline project_report/tex/design.tex:3]: Maybe add a figure, if page space allows it.

**Review by Kechr — CHANGES_REQUESTED** [review]

File: project_report/tex/design.tex (hunk 4 above)
Comment (Bokaiii) [reply project_report/tex/design.tex:16]: Not really sure what you want me to change?

**Comment by Kechr** [comment]
I think you misunderstood my previous comment. 

Try reading your sentence:
"This pattern separates the database (data-related logic), the GUI (UI logic) and the logic (interface between the two) into different layers, and thereby helps keep separation of concerns."

Does this make sense? Is the database a part of the dashboard? Is the logic an interface? What happened to Model, View and Controller - isn't that what MVC is all about? 

[Try rereading the theory about MVC](https://www.tutorialspoint.com/mvc_framework/mvc_framework_introduction.htm) and make the necessary changes.

**Comment by Kechr** [comment]
> 
> 
> I think you misunderstood my previous comment.
> 
> Try reading your sentence:
> "This pattern separates the database (data-related logic), the GUI (UI logic) and the logic (interface between the two) into different layers, and thereby helps keep separation of concerns."
> 
> Does this make sense? Is the database a part of the dashboard? Is the logic an interface? What happened to Model, View and Controller - isn't that what MVC is all about?
> 
> [Try rereading the theory about MVC](https://www.tutorialspoint.com/mvc_framework/mvc_framework_introduction.htm) and make the necessary changes.

I'm sorry, but this still applies.

File: project_report/tex/design.tex (hunk 4 above)
Comment (Bokaiii) [reply project_report/tex/design.tex]: Not the controller - actual interfaces

File: project_report/tex/design.tex [hunk 6]
```diff
@@ -1 +1,25 @@
...
+the model, to retrieve or store data, or the view, to present data to the user.
+
+The group has decided to use the MVC design pattern, thus making it possible to
+work on the different aspects of the system (GUI, database, logic) separately.
+To ensure that these different parts of the system can communicate, a
+'contract', should be created.
+
+A visualisation of the MVC pattern can be seen in figure \ref{figure:MVC_model}
+
+\begin{figure}[ht]
```
Comment (Kechr) [inline project_report/tex/design.tex:20]: If we didn't make the diagram, we either need to make one ourselves or throw a source in for the one we use.

**Review by Kechr — CHANGES_REQUESTED** [review]
Looks good otherwise :)

**Review by Kechr — APPROVED** [review]

**Pull request merged**